### DISCUSSION TITLE
Voting says "You must be logged in to vote"

### DISCUSSION THREAD

**Discussion by karliss** [community/community#3059] [Discussions] [answered]
Just tried enabling discussions in a test repository. When I open an issue there is an upvote button but clicking it shows "You must be logged in to vote " even though I am logged in. I asked others to try and they got the same error. I also notice that my dicussion list doesn't show upvote button while discussion list in this repository shows them.

Here is my test repo https://github.com/karliss/github-integration-test/discussions/9

Upvote button shown
![image](https://user-images.githubusercontent.com/7101031/103090049-9a6d6a80-45f8-11eb-806c-fafc8bcd0dcf.png)
Upvote button missing in the list:
![image](https://user-images.githubusercontent.com/7101031/103090082-b53fdf00-45f8-11eb-9427-93dc5d67c7ef.png)
Upvote button visible in discussion list for this repo:
![image](https://user-images.githubusercontent.com/7101031/103090144-db657f00-45f8-11eb-8949-2b278084f7df.png)

**Comment by ITAYC0HEN** [comment]
I experience the same issues. On the "Discussions" on some repostiories I can't upvote and get an error "You must be logged in to vote". Moreover, the Discussions on some repositories allow to upvote comments, while in others they are not.

**Comment by shaderecker** [comment]
I experienced the same issues here: https://github.com/telekom-security/tpotce/discussions/742

**Reply by shaderecker** [reply]
Funnily here in https://github.com/github/feedback it works correctly

**Comment by andria-dev** [comment]
I get the same message when trying to upvote my own discussion (wasn't obvious this was an upvote feature lol).

**Comment by rotimi-best** [comment]
Got the same issue

![github-upvote-bug](https://user-images.githubusercontent.com/31730715/103183318-45f91200-48ba-11eb-932c-4123f0adeba6.png)

**Comment by andreynering** [comment]
The same here

https://github.com/go-task/task/discussions/422

![Firefox_Screenshot_2020-12-28T12-15-31 960Z](https://user-images.githubusercontent.com/7011819/103213897-d4d35200-48ed-11eb-8c90-b3d153da5f65.png)

**Comment by becca** [comment]
Thank you for all of this feedback. We're still refining the voting experience within Discussions - it's an early feature that isn't rolled out yet to everyone in the Discussions public beta. We will resolve the noted issues here as soon as possible!

**Comment by vsoch** [comment]
Just hit the same issue!

![image](https://user-images.githubusercontent.com/814322/103387549-a7d79880-4ac1-11eb-8e64-826bfd7fc605.png)

Thanks for working on this @becca we're all really excited!

**Comment by evi-liu** [comment, marked as answer]
Hi all, this bug should be fixed now! Please let me know if you continue to see it, and we can investigate further.

**Reply by vsoch** [reply]
Thank you @evi-liu! Just to confirm - we shouldn't see anything now? For me the voting icon is totally gone. 

![image](https://user-images.githubusercontent.com/814322/103587443-7df8ea00-4ea4-11eb-895e-289aad3e8d81.png)

I do hope that we can upvote in the future, it will make discussions comparable to Stack Overflow (which is what several of my communities would hugely benefit from!)

**Reply by evi-liu** [reply]
Yes, we're still beta testing upvoting to a limited group, but it'll be available to all Discussions repositories soon!

**Reply by ghost** [reply]
Still doesnt work on mobile stay mad
